packages/management-console/server/server.js

```diff
@@ -730,7 +730,12 @@ const resolvers = {
                 console.log("data", datum.parentProcessInstanceId);
                 if (args["filter"].id) {
                   return (datum.id == args["filter"].id);
-                } else {
+                }
+                else if (args["filter"].parentProcessInstanceId[0] == null){
```

Comment:
please align else statements
```suggestion
               } else if (args["filter"].parentProcessInstanceId[0] == null){
```

Reply:
Should an else statement be addded?

Reply:
its just the formatting that looks a bit odd.

Reply:
I guess I will as the prettier soon :) . That should solve most of the formatting issue.

Reply:
you could just fix these ones quite easily, the prettier formatting can be done later